mlflow/server/js/src/common/utils/FeatureUtils.ts

```diff
@@ -6,41 +6,43 @@
  * Disable legacy charts on "compare runs" page in favor of a new chart view
  */
 export const shouldDisableLegacyRunCompareCharts = () => false;
+
 /**
- * UI feature preview: displays data lineage (datasets used) in experiment runs
+ * Enables features related to deep learning: Phase 3
+ * Includes improved runs visibility controls, reworked hover tooltip and improved handling of charts on run details page.
  */
-export const shouldEnableExperimentDatasetTracking = () => true;
+export const shouldEnableDeepLearningUIPhase3 = () => false;
```

Comment:
I think we can enable this one already, WDYT @daniellok-db ?

Reply:
yeah let's do it